### PR TITLE
homework 11

### PR DESCRIPTION
**Pull request by artsiom-tsaryonau**

### CODE REVIEW

File: homework11/build.gradle [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+bootRun {
+    jvmArgs = [
+            "-XX:NewSize=1m",
+            "-XX:MaxNewSize=5m",
+            "-Xms64m"
+    ]
+}
+
```
Comment (AShynkevich) [inline homework11/build.gradle:23]: remove the line

File: homework11/src/main/java/hello/HelloController.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@RestController
+public class HelloController {
+
+    @RequestMapping("/")
+    public String index() {
+        return "Greetings from Spring Boot!";
+    }
+
```
Comment (AShynkevich) [inline homework11/src/main/java/hello/HelloController.java]: remove the line

File: homework11/src/main/java/hello/Application.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                System.out.println(beanName);
+            }
+
+            List<String> beans = ManagementFactory.getGarbageCollectorMXBeans().stream()
+                    .map(GarbageCollectorMXBean::getName)
+                    .collect(Collectors.toList());
+            System.out.println("Garbage collectors: " + beans);
+        };
+    }
+
```
Comment (AShynkevich) [inline homework11/src/main/java/hello/Application.java]: remove the line

File: homework11/build.gradle (hunk 1 above)
Comment (artsiom-tsaryonau) [reply homework11/build.gradle:23]: removed

**Pull request merged**